Source/WebCore/platform/graphics/transforms/TransformationMatrix.cpp

```diff
@@ -907,6 +908,13 @@ TransformationMatrix& TransformationMatrix::scale3d(double sx, double sy, double
     return *this;
 }
 
+static double roundEpsilonToZero(double val)
+{
+    if (-FLT_EPSILON < val && val < FLT_EPSILON)
```

Comment:
For `double` I believe we want [`DBL_EPSILON`](https://en.cppreference.com/w/c/types/limits#:~:text=FLT_EPSILON-,DBL_EPSILON,-LDBL_EPSILON)?